remoteworkitem/flatten_test.go

```diff
@@ -0,0 +1,34 @@
+package remoteworkitem
```

Comment:
You just need to update the call to Flatten with full package name; remoteworkitem.Flatten